frontend/lib/src/components/shared/StreamlitMarkdown/styled-components.ts

```diff
@@ -209,6 +212,14 @@ export const StyledStreamlitMarkdown =
           borderLeft: `${theme.sizes.borderWidth} solid ${theme.colors.lightGray}`,
         },
 
+        "b, strong": {
+          fontWeight: theme.fontWeights.bold,
```

Comment:
I figured that copy & paste into rich text editors such as Google Docs does not preserve the boldness if it is a number and only if the value is `bold`. Nothing for the PR here but good to keep in mind for the future.

Reply:
Yeah, I tried to figure out if there is a good fix for using `bold` & using a smaller weight... but I couldn't find something. Also checked Notion, and if you copy out bold text it seems to have the same issue. At least for `st.html` it should work fine now.